test/unit/specs/date-picker.spec.js

```diff
@@ -1272,20 +1272,23 @@ describe('DatePicker', () => {
           triggerEvent(leftCell, 'click', true);
           setTimeout(_ => {
             triggerEvent(rightCell, 'mousemove', true);
-            triggerEvent(rightCell, 'click', true);
-
             setTimeout(_ => {
-              const {
-                minDate,
-                maxDate
-              } = vmWithDefaultTime.picker;
-              expect(minDate.getHours()).to.be.equal(11);
-              expect(minDate.getMinutes()).to.be.equal(59);
-              expect(minDate.getSeconds()).to.be.equal(59);
-              expect(maxDate.getHours()).to.be.equal(0);
-              expect(maxDate.getMinutes()).to.be.equal(0);
-              expect(maxDate.getSeconds()).to.be.equal(0);
-              done();
+              expect(rightCell.className.split(' ').indexOf('in-range') >= 0).to.be.true;
```

Comment:
How about using `classList` ?

Reply:
I'm not sure about the browser support requirement of element. But it is safer to use `className` and almost the same complex.

Reply:
Tests are executed in Chrome. so I don't think this will be an issue.

Reply:
Maybe possible future e2e tests in different browsers? If browser supporting has never become an issue when writing test cases in this project, I'll use `classList.contains` instead.

Reply:
You can safely use classList. 2.0 requires ie >10 which supports it.  see https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

Reply:
great.